### PR TITLE
Ocd 2789 - Improve Reports responsiveness - page load - User Activity

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: PENDING_RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 
-## Version 10.19.0
+## Version 10.20.0
 _Date TBD_
 
 ### New features
 * Use new endpoints for better ONC-ACB and ONC-ATL report activity
```
Comment (kekey1) [inline PENDING_RELEASE_NOTES.md]: This was already released

File: PENDING_RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 * Add link to FF4j admin console for ROLE_ADMIN
+* Updated user and user action activity reports to use activity metadata
 
 ## Bug Fixes
 * Allow upload of files from Edge
```
Comment (kekey1) [inline PENDING_RELEASE_NOTES.md]: This was already released

File: src/app/pages/reports/userActions/userActions.component.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        search () {
+            this.initializeSearch();
+            this.callActivityServiceForMetadata('listings');
+            this.callActivityServiceForMetadata('products');
+            this.callActivityServiceForMetadata('versions');
+            this.callActivityServiceForMetadata('users');
+            this.callActivityServiceForMetadata('announcements');
+            this.callActivityServiceForMetadata('developers');
+            this.callActivityServiceForMetadata('pending_listings');
+            this.callActivityServiceForMetadata('correct_action_plans');
```
Comment (kekey1) [inline src/app/pages/reports/userActions/userActions.component.js]: noticed this a couple times in the API too... the term is "corrective action plan" not "correct", not a big deal but probably worth fixing to keep our language consistent

File: PENDING_RELEASE_NOTES.md (hunk 1 above)
Comment (tmy1313) [reply PENDING_RELEASE_NOTES.md]: Removed.

File: src/app/navigation/navigation-top.html [hunk 4]
```diff
@@ -96,7 +96,8 @@
             <li><a ui-sref="reports.products()">Products</a></li>
             <li><a ui-sref="reports.versions()">Versions</a></li>
             <li ng-if="vm.hasAnyRole(['ROLE_ADMIN', 'ROLE_ONC'])"><a href="#/admin/reports/announcement">Announcements</a></li>
-            <li ng-if="vm.hasAnyRole(['ROLE_ADMIN', 'ROLE_ONC'])"><a href="#/admin/reports/users">Users</a></li>
+            <li><a ui-sref="reports.users()">Users</a></li>
```
Comment (andlar) [inline src/app/navigation/navigation-top.html]: These need to be restricted to only ROLE_ADMIN and ROLE_ONC. ROLE_ACB should not be able to see these

**Pull request merged**